blocks/network-link/network-link.php

```diff
@@ -0,0 +1,137 @@
+<?php
+/**
+ * Network Link
+ * Server-side rendering of the `cata/network-link` block.
+ *
+ * @package Cata\Blocks
+ */
+
+namespace Cata\Blocks;
+
+use WP_Block;
+
+/**
+ * Register Network Link block
+ */
+function register_network_link_block() {
+	if ( ! apply_filters( 'cata_blocks_support_network_link_block', true ) ) {
+		return;
+	}
+	register_block_type( 
+		__DIR__ . '/build',
+		array(
+			'render_callback' => __NAMESPACE__ . '\\render_block_cata_network_link',
+		)
+	);
+}
+add_action( 'init', __NAMESPACE__ . '\\register_network_link_block' );
+
+/**
+ * Renders the `cata/network-link` block on server.
+ *
+ * @param array    $attributes The block attributes.
+ * @param string   $content    InnerBlocks content of the Block.
+ * @param WP_Block $block      Block object.
+ *
+ * @return string Rendered HTML of the referenced block.
+ */
+function render_block_cata_network_link( array $attributes, string $content, WP_Block $block ) : string {
+	$service = ( isset( $attributes['service'] ) ) ? $attributes['service'] : 'Icon';
+	$url     = ( isset( $attributes['url'] ) ) ? $attributes['url'] : false;
+	$label   = ( isset( $attributes['label'] ) ) ? $attributes['label'] : block_cata_network_link_get_name( $service );
+	$width   = ( isset( $attributes['width'] ) ) ? $attributes['width'] : 'auto';
+
+	if ( ! $url ) {
+		return '';
+	}
+
+	$link  = '<li class="cata-network-link cata-network-link-' . $service . '" style="inline-size:' . esc_attr( $width ) . ';">';
+	$link .= '<a href="' . esc_url( $url ) . '" class="wp-block-cata-network-link-anchor">';
+	$link .= block_cata_network_link_get_icon( $service );
+	$link .= '<span class="wp-block-cata-network-link-label screen-reader-text">';
```

Comment:
When including screen reader text, this article and the others it cites at the bottom suggest we use attributes to hide the SVG itself from screen readers. https://css-tricks.com/accessible-svg-icons/

Reply:
I removed the screen reader text and added an `aria-label` attribute to the anchor tag like the article suggested. I also added `aria-hidden` to the SVGs